### PR TITLE
The test verifies directory export code path with import and export killpoints.

### PR DESCRIPTION
**Pull request by vishalkanaujia**
Fixes: http://tracker.ceph.com/issues/17835
Signed-off-by: Vishal Kanaujia vishal.kanaujia@sandisk.com

Revised PR of https://github.com/ceph/ceph/pull/12881

### CODE REVIEW

**Comment by jcsp** [comment]
Is this passing for you?  I'm seeing failures:
http://pulpito.ceph.com/jspray-2017-02-15_00:08:14-fs-wip-jcsp-testing-20170214b-distro-basic-smithi/815863
http://pulpito.ceph.com/jspray-2017-02-15_00:08:14-fs-wip-jcsp-testing-20170214b-distro-basic-smithi/815955

**Comment by vishalkanaujia** [comment]
@jcsp The failure was in the MDS restart. After a MDS hit a kill point, we restart it and continue the test. However, MDS restart could not happen and wait timed out.

2017-02-15T02:41:53.325 INFO:tasks.ceph.mds.a:Restarting daemon
2017-02-15T02:41:53.325 INFO:tasks.ceph.mds.a:Stopping old one...
2017-02-15T02:41:53.325 DEBUG:tasks.ceph.mds.a:waiting for process to exit
2017-02-15T02:41:53.326 INFO:teuthology.orchestra.run:waiting for 300
2017-02-15T02:41:53.326 INFO:tasks.cephfs_test_runner:test_export (tasks.cephfs.test_export.TestKillPoints) ... ERROR

I made change to fail the MDS before restarting. I also updated all kill points count to iterate  for import and export.

In my setup, the test ran successfully.

**Comment by jcsp** [comment]
Last time I ran this it was still failing: http://pulpito.ceph.com/jspray-2017-02-22_15:15:03-fs-wip-jcsp-testing-20170222-distro-basic-smithi/847142

Rebase also needed.

**Comment by batrick** [comment]
@lesserevil I'm taking a look at this now. I'll have a few changes I'll send you PRs against this branch.

**Comment by batrick** [comment]
@vishalkanaujia please rebase on master when you have time.

File: qa/suites/fs/recovery/tasks/export.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (amitkumar50) [inline qa/suites/fs/recovery/tasks/export.yaml]: @vishalkanaujia  Please consider to make title < 50 character of PR
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#title-of-pull-requests-and-title-of-commits

Comment (amitkumar50) [inline qa/suites/fs/recovery/tasks/export.yaml]: @vishalkanaujia 'It is required to prefix it with the subsystem or module you are changing'

**Comment by jcsp** [comment]
Wow, this one is ancient, but fundamentally still desirable if anyone else wanted to pick it up

**Pull request closed without merging**